source/client/benchmark_client_impl.cc

```diff
@@ -189,16 +189,18 @@ void BenchmarkClientHttpImpl::setRequestHeader(absl::string_view key, absl::stri
 }
 
 bool BenchmarkClientHttpImpl::tryStartOne(std::function<void()> caller_completion_callback) {
+  if (!cluster_->resourceManager(Envoy::Upstream::ResourcePriority::Default)
```

Comment:
Not sure I see how this diff relates to the PR description..

Reply:
Without this change, we would see the overflow stats rise excessively. With it we  will end up reporting the time we spend not being able to queue u a new request as time spend blocking (we indicate we could not by returning false). Hope that clarifies this?

Reply:
I think so, but can you add comments and a test?
/wait

Reply:
Done, and done. The test is implemented with python, wdyt?